duckduckgo/Android
# Update widget text to "Search DuckDuckGo" as per user feedback

Task/Issue URL: https://app.asana.com/0/inbox/361428290920650/1126670379131263/1126674827039006
Tech Design URL: N/A

**Description**:
Update widget text back to "Search DuckDuckGo" after feedback from users that they did not like "Search Privately"

**Steps to test this PR**:
1. Add widget to home screen
1. Ensure preview images of widgets show "Search DuckDuckGo"
1. Ensure widget text also reads "Search DuckDuckGo"

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
